lib/vfscore/automount.c

```diff
@@ -346,10 +350,86 @@ extern const char vfscore_einitrd_start[];
 extern const char vfscore_einitrd_end;
 #endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_EINITRD */
 
+#define LIBVFSCORE_EXTRACTOPT_MOUNT_RAMFS		(0x1 << 0)
+#define LIBVFSCORE_EXTRACTOPT_MOUNT_IFNOMOUNT		(0x1 << 1)
+#define LIBVFSCORE_EXTRACTOPT_FORMAT_AUTO		(0x1 << 2)
+
+static inline int vfscore_extract_parse_opts(const char *opts)
+{
+	char strbuf[64];
```

Comment:
You know I am going to say it... :triumph:  **Reverse Xmas Tree**? :pleading_face: Also s/size_t/__sz, s/ssize_t/__ssz.?

Reply:
Data types, I agree, will adopt. Reverse xmas..., I find this optional. but I'll change it for you.

Reply:
We did not use any Unikraft internal data types in this file, I'll keep `size_t` and co.